### PR TITLE
Some issues were fixed

### PR DESCRIPTION
**Pull request by MariaMaximova**
https://github.com/MariaMaximova/convert_temp/issues/2 setup.py file was added + some correction in file .gitignore was made.
https://github.com/MariaMaximova/convert_temp/issues/3 folder converter + file converter.py was added. Logic was imported in main.py and some local tests were passed.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,31 +1,12 @@
 import argparse
+from converter.converter import *
 
 
 CELSIUM_INPUT = ['celsius', 'cels', 'cel', '-c', 'c']
 FAH_INPUT = ['fahrenheit', 'fahren', 'far', '-f', 'f']
 QUIT_INPUT = ['quit', 'q']
```
Comment (OleksiiBulba) [inline main.py:7]: @MariaMaximova, I have suggession to add `exit` here, just in case

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+with open("Readme.md", "r") as readme_file:
+    readme = readme_file.read()
+
+setup(
+    name="converter",
+    version="1.0.1",
```
Comment (OleksiiBulba) [inline setup.py:8]: @MariaMaximova, from one hand it's ok to add plain version to setup.py file. From the other, I think I will suggest you adding a file `version.txt` and loading version from it as you load readme from Readme.md file.
When you will update version, you will need update only version.txt file.

Comment (OleksiiBulba) [reply setup.py:8]: Also, I would suggest you working not directly in `dev` branch but in separate feature branch, for example, for this MR you could create a new branch `issue-1-fix-add-setup-file` or `fix-issues-1-and-2`. Then create MR from that branch to `dev` branch (not changing the version). And then you can create release MR from `dev` to `main` branch. That's like a little simplified [github workflow](https://quangnguyennd.medium.com/git-flow-vs-github-flow-620c922b2cbd) when you combine master and release branches together.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    
+    author="Maria Maksimova",
+    author_email="maryia.maksimava@yahoo.com",
+    
+    description="A package to converting temperature from Celsius to Fahrenheit and vise versa",
```
Comment (OleksiiBulba) [inline setup.py:13]:
```suggestion
    description="A package for converting temperature from Celsius to Fahrenheit and vise versa",
```

File: setup.py (hunk 2 above)
Comment (MariaMaximova) [reply setup.py]: Added in [setup.py](https://github.com/MariaMaximova/convert_temp/commit/67f37cda14e8921dad8a4e3e96f9f8abb4ad93d8)

File: setup.py (hunk 3 above)
Comment (MariaMaximova) [reply setup.py]: Added in [setup.py](https://github.com/MariaMaximova/convert_temp/commit/67f37cda14e8921dad8a4e3e96f9f8abb4ad93d8)

File: main.py (hunk 1 above)
Comment (MariaMaximova) [reply main.py]: Added in [main.py](https://github.com/MariaMaximova/convert_temp/commit/67f37cda14e8921dad8a4e3e96f9f8abb4ad93d8)

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
+from setuptools import setup, find_packages
```
Comment (OleksiiBulba) [inline setup.py:1]: Is `find_packages` somehow used? Or can we remove it?

**Review by OleksiiBulba — APPROVED** [review]

**Pull request merged**